modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -589,35 +726,68 @@ public CompletableFuture<BinaryRow> get(
         );
     }
 
+    /**
+     * Checks that the batch of rows belongs to one partition.
+     *
+     * @param rows Batch of rows.
+     * @return If all rows belong to one partition, the method returns true; otherwise, it returns false.
+     */
+    private boolean isOnePartitionBatch(Collection<BinaryRowEx> rows) {
```

Comment:
It's better to use Single instead of One here. Same in javadoc.

Reply:
Done.